src/i18n/locales/en-US.json

```diff
@@ -309,7 +309,7 @@
   "global.actions.dialogs.disableEasyConfirmationFirst.title": "Action failed",
   "global.actions.dialogs.enableFingerprintsFirst.message": "You need to enable biometrics in your device first in order to be able link it with this app",
   "global.actions.dialogs.enableFingerprintsFirst.title": "Action failed",
-  "global.actions.dialogs.enableSystemAuthFirst.message": "You probably disabled lock screen in your phone. You need to disable easy transaction confirmation first. Please set up you lock screen (PIN / Password / Pattern) on your phone and then restart application. After this action you should be able to disable lock screen on your phone and use this application",
+  "global.actions.dialogs.enableSystemAuthFirst.message": "You probably disabled the lock screen on your phone. You need to disable easy transaction confirmation first. Please set up you lock screen (PIN / Password / Pattern) on your phone and then restart application. After this action you should be able to disable the lock screen on your phone and use this application",
```

Comment:
This sentence doesn't really make sense to me. I assume it's talking about the app's lock screen and not the phone's native lock screen?

Reply:
Nope, they were referring to the app lock screen. You need to activate it in your System preferences, and *then* you'll be able to use easy confirmation.